sources/Inputs/src/NetworkManager.cpp

```diff
@@ -55,68 +56,78 @@ NetworkManager::buildTree() {
   auto opt_id = network->getSlackNodeBusId();
 
   const auto& voltageLevels = network->getVoltageLevels();
-  for (auto it_v = voltageLevels.begin(); it_v != voltageLevels.end(); ++it_v) {
-    auto vl = std::make_shared<VoltageLevel>((*it_v)->getID());
+  for (const auto& networkVL : voltageLevels) {
```

Comment:
Can we please have the update of the DFL database in another PR?

Reply:
I don't get why it is relevant to this line

Reply:
I'll re do the PR with only the setting / assembling file in it after validation